packages/react/src/views/EmbeddedChat.js

```diff
@@ -178,6 +201,10 @@ const EmbeddedChat = ({
     [RCInstance, ECOptions]
   );
 
+  if (!useRemoteProps(remoteOpt, RCInstance, setConfig)) {
```

Comment:
we shouldn't use hooks like this

Reply:
Check now, if this looks better ?

Reply:
i feel like the hook to be redundant, it can be simplified if we just add a useEffect and check if (remoteOpt == true) {  } then we would not need to pass the RCInstance or a setter to a hook

Reply:
Yep sure, that can be done, i just made it like this to reduce the amount of code in EmbeddedChat main component.. and keep it clean.. I'll change the structure.

Reply:
Done!